cdap-master/pom.xml

```diff
@@ -298,7 +298,7 @@
                   <goal>copy-resources</goal>
                 </goals>
                 <configuration combine.self="override">
-                  <outputDirectory>${stage.artifacts.dir}/spark1_2.10</outputDirectory>
+                  <outputDirectory>${stage.artifacts.dir}/${spark1.artifacts.dir}</outputDirectory>
```

Comment:
call it as `spark1_version`

Reply:
there's already a property called `spark1.version`

Reply:
then why can't you use that.

Reply:
basically the value is not directory specific

Reply:
will changing the name of these directories break backward compatibility?

Reply:
never-mind. I just meant that the value isn't specific to directory, but the usage seems only directory specific, so the name can remain the way it is.